lib/public_api_v1.ts

```diff
@@ -38,6 +39,9 @@ const app = express();
 const COSMOSDB_URI: string = process.env.CUSTOMCONNSTR_COSMOSDB_URI;
 const COSMOSDB_KEY: string = process.env.CUSTOMCONNSTR_COSMOSDB_KEY;
 
+const MESSAGE_CONTAINER_NAME: NonEmptyString =
```

Comment:
same here you don't know whether `process.env.MESSAGE_CONTAINER_NAME` is empty or not, you should go through `isNonEmptyString` or `toNonEmptyString`